tests/integration/features/bootstrap/FeatureContext.php

```diff
@@ -2326,6 +2332,47 @@ public function userCreatesBreakoutRooms(string $user, int $amount, string $mode
 		$this->assertStatusCode($this->response, $status);
 	}
 
+	/**
+	 * @Then /^user "([^"]*)" broadcasts message "([^"]*)" to room "([^"]*)" with (\d+)(?: \((v1)\))?$/
+	 *
+	 * @param string $user
+	 * @param string $message
+	 * @param string $identifier
+	 * @param string $statusCode
+	 * @param string $apiVersion
+	 */
+	public function userBroadcastsMessageToBreakoutRooms(string $user, string $message, string $identifier, string $statusCode, string $apiVersion = 'v1') {
+		$body = new TableNode([['message', $message]]);
+
+		$this->setCurrentUser($user);
+		$this->sendRequest(
+			'POST', '/apps/spreed/api/' . $apiVersion . '/breakout-rooms/' . self::$identifierToToken[$identifier] . '/broadcast',
+			$body
+		);
+
+		$this->assertStatusCode($this->response, $statusCode);
+		sleep(1); // make sure Postgres manages the order of the messages
```

Comment:
Yes, postgres has some issues when multiple auto increment occur in the same second. That is why we also sleep when posting a message, otherwise the chat order can be mixed up:

```
Targets
    Occurrences of 'sleep(1); // make sure Postgres manages the order of the messages' in Project
Found Occurrences in Project  (5 usages found)
    Usage in comments  (5 usages found)
        Nextcloud 26  (5 usages found)
            apps/spreed/tests/integration/features/bootstrap  (5 usages found)
                FeatureContext.php  (5 usages found)
                    1541 sleep(1); // make sure Postgres manages the order of the messages
                    1572 sleep(1); // make sure Postgres manages the order of the messages
                    1902 sleep(1); // make sure Postgres manages the order of the messages
                    1931 sleep(1); // make sure Postgres manages the order of the messages
                    2363 sleep(1); // make sure Postgres manages the order of the messages
```